packages/wiz/data_stream/vulnerability/elasticsearch/ingest_pipeline/default.yml

```
@@ -585,9 +538,15 @@ processors:
       ignore_empty_value: true
   - set:
       field: resource.id
-      tag: set_resource_id
+      tag: set_resource_id_from_provider_unique_id
       copy_from: wiz.vulnerability.vulnerable_asset.provider_unique_id
       ignore_empty_value: true
+  - set:
+      field: resource.id
+      tag: set_resource_id_from_resource_id
```

```suggestion
      tag: set_resource_id_from_vulnerable_asset_id
```
To give better context.